handler/src/main/java/io/netty/handler/ssl/JdkSslServerContext.java

```diff
@@ -214,17 +221,19 @@ public JdkSslServerContext(File trustCertCollectionFile, TrustManagerFactory tru
      * @param sessionCacheSize the size of the cache used for storing SSL session objects.
      *                         {@code 0} to use the default value.
      * @param sessionTimeout the timeout for the cached SSL session objects, in seconds.
-     *                       {@code 0} to use the default value.
+     *                         {@code 0} to use the default value
      * @deprecated use {@link SslContextBuilder}
      */
     @Deprecated
     public JdkSslServerContext(File trustCertCollectionFile, TrustManagerFactory trustManagerFactory,
-            File keyCertChainFile, File keyFile, String keyPassword, KeyManagerFactory keyManagerFactory,
-            Iterable<String> ciphers, CipherSuiteFilter cipherFilter, JdkApplicationProtocolNegotiator apn,
-            long sessionCacheSize, long sessionTimeout) throws SSLException {
+                               File keyCertChainFile, File keyFile,
```

Comment:
FYI: it's really annoying to see these whitespace changes for something that otherwise didn't change and it wasn't previously incorrect. In the future, try to avoid such changes.